app/assets/javascripts/es6-components.js

```diff
@@ -0,0 +1,13 @@
+// These modules from govuk_publishing_components
+// depend on govuk-frontend modules. govuk-frontend
+// now targets browsers that support `type="module"`.
+//
+// To gracefully prevent execution of these scripts
+// on browsers that don't support ES6, this script
+// should be included in a `type="module"` script tag
+// which will ensure they are never loaded.
+
+//= require govuk_publishing_components/components/character-count
+//= require govuk_publishing_components/components/error-summary
+//= require govuk_publishing_components/components/radio
+//= require govuk_publishing_components/components/tabs
```

Comment:
Where did tabs come from? Is this needed?

Reply:
Thanks Andy, not sure where they came from, but I have removed, as it is not required in the feedback app.